src/components/SEO.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+import PropTypes from 'prop-types'
+import { StaticQuery, graphql } from 'gatsby'
+
+const query = graphql`
+  query GetSiteMetadata {
+    site {
+      siteMetadata {
+        title
+        author
+        description
+        siteUrl
+        social {
+          twitter
+        }
+      }
+    }
+  }
+`
+
+function SEO({ meta, image, title, description }) {
```

Comment:
See [this guide](https://www.gatsbyjs.org/docs/add-seo-component/) for more info. Reusability here is pretty great, and it's nice to default some fields.